datastore/google/cloud/datastore/client.py

```diff
@@ -456,19 +456,13 @@ def query(self, **kwargs):
           >>> query = client.query(kind='MyKind')
           >>> query.add_filter('property', '=', 'val')
 
-        Using the query iterator's
-        :meth:`~google.cloud.datastore.query.Iterator.next_page` method:
+        Using the query iterator
```

Comment:
Since using cursors is a very common pattern in datastore, can you add a quick sample here showing how to accomplish that?